src/renderer/store/editor.js

```diff
@@ -681,6 +681,32 @@ const actions = {
     }
   },
 
+  CLOSE_OTHER_TABS ({ state, dispatch }, file) {
+    const { tabs } = state
+    tabs.filter(f => f.id !== file.id).forEach(tab => {
+      dispatch('CLOSE_TAB', tab)
```

Comment:
We have `CLOSE_TABS` action, so no need to `dispatch('CLOSE_TAG', tab)` each time?

Reply:
@Jocs `CLOSE_TABS` mutation forces to close all unsaved tabs without save prompt. So, if we dispatch `CLOSE_TABS` once, all unsaved tabs will be lost.